frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.scss

```diff
@@ -0,0 +1,174 @@
+@import "./../../../assets/styles/variables.scss";
+@import "./../../../assets/styles/common.scss";
+
+.shukra-right-container {
```

Comment:
Usually when I see repeated prefixes like `.shukra-`, it's because the project style is based more on the trello approach where there isn't any nesting and each classname is at the root level of the sheet but I don't think that's the case here (unless there's some webpack kind of logic which combines classnames together before render)

But if they are being nested I don't see the need for the `.shukra-` prefix on every class.